rotkehlchen/accounting/structures/base.py

```diff
@@ -169,7 +169,10 @@ def deserialize_from_db(cls, entry: HISTORY_EVENT_DB_TUPLE_READ) -> 'HistoryBase
     @property
     def serialized_event_identifier(self) -> str:
         """Take a HistoryBaseEntry's event_identifier and returns a string representation."""
-        if self.location == Location.KRAKEN:
+        # The only cases where this is used are in Kraken & Rotki generic events imports.
+        # For rotki events import ids, uuid4().hex is used which has a length of 32,
+        # hence the condition.
+        if self.location == Location.KRAKEN or len(self.event_identifier.decode()) == 32:
```

Comment:
This feels like something that can break in the future. What if instead you prepepend something like `rotki_imported` and  then you can do `self.event_identifier.startswith(b'rotki_imported')`